src/Dialogs/ColorPreferencesDialog.vala

```diff
@@ -0,0 +1,403 @@
+            color08_button = new Gtk.ColorButton ();
+            color09_button = new Gtk.ColorButton ();
+            color10_button = new Gtk.ColorButton ();
+            color11_button = new Gtk.ColorButton ();
+            color12_button = new Gtk.ColorButton ();
+            color13_button = new Gtk.ColorButton ();
+            color14_button = new Gtk.ColorButton ();
+            color15_button = new Gtk.ColorButton ();
+            color16_button = new Gtk.ColorButton ();
+            background_button = new Gtk.ColorButton ();
+            background_button.use_alpha = true;
+            foreground_button = new Gtk.ColorButton ();
+            cursor_button = new Gtk.ColorButton ();
+            cursor_button.use_alpha = true;
+
+            color01_button.color_set.connect (update_settings_from_buttons);
+            color02_button.color_set.connect (update_settings_from_buttons);
+            color03_button.color_set.connect (update_settings_from_buttons);
+            color04_button.color_set.connect (update_settings_from_buttons);
+            color05_button.color_set.connect (update_settings_from_buttons);
+            color06_button.color_set.connect (update_settings_from_buttons);
+            color07_button.color_set.connect (update_settings_from_buttons);
+            color08_button.color_set.connect (update_settings_from_buttons);
+            color09_button.color_set.connect (update_settings_from_buttons);
+            color10_button.color_set.connect (update_settings_from_buttons);
+            color11_button.color_set.connect (update_settings_from_buttons);
+            color12_button.color_set.connect (update_settings_from_buttons);
+            color13_button.color_set.connect (update_settings_from_buttons);
+            color14_button.color_set.connect (update_settings_from_buttons);
+            color15_button.color_set.connect (update_settings_from_buttons);
+            color16_button.color_set.connect (update_settings_from_buttons);
+            background_button.color_set.connect (update_settings_from_buttons);
+            foreground_button.color_set.connect (update_settings_from_buttons);
+            cursor_button.color_set.connect (update_settings_from_buttons);
+
+            var contrast_box = new Gtk.Box (Gtk.Orientation.VERTICAL, 3);
+            var contrast_top_label = new Gtk.Label ("┐") {
+                halign = Gtk.Align.START,
+                margin_start = 9
+            };
+            contrast_image = new Gtk.Image.from_icon_name ("process-completed", Gtk.IconSize.LARGE_TOOLBAR) {
+                halign = Gtk.Align.START
+            };
+            var contrast_bottom_label = new Gtk.Label ("┘") {
+                halign = Gtk.Align.START,
+                margin_start = 9
+            };
+            contrast_box.pack_start (contrast_top_label);
+            contrast_box.pack_start (contrast_image);
+            contrast_box.pack_start (contrast_bottom_label);
```

Comment:
It seems like it would be easier to do:
```suggestion
            var contrast_top_label = new Gtk.Label ("┐");
            contrast_image = new Gtk.Image.from_icon_name ("process-completed", Gtk.IconSize.LARGE_TOOLBAR);
            var contrast_bottom_label = new Gtk.Label ("┘");
            
            var contrast_box = new Gtk.Box (Gtk.Orientation.VERTICAL, 3) {
                  halign = Gtk.Align.START
            };
            contrast_box.pack_start (contrast_top_label);
            contrast_box.pack_start (contrast_image);
            contrast_box.pack_start (contrast_bottom_label);
```

Reply:
Definitely.